bin/glow-monitor/monitor-udp.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Define the path for the timestamp file and log file
+TIMESTAMP_FILE="/dev/shm/last-udp.txt"
+
+# Ensure the timestamp file exists and has the current timestamp
+touch "$TIMESTAMP_FILE"
+echo "$(date +%s)" > "$TIMESTAMP_FILE"
+
+# Function to update timestamp
+update_timestamp() {
+    echo "$(date +%s)" > "$TIMESTAMP_FILE"
+}
+
+# Function to check the age of the timestamp
+check_timestamp() {
+    current_time=$(date +%s)
+    last_timestamp=$(cat "$TIMESTAMP_FILE")
+    time_diff=$((current_time - last_timestamp))
+
+    # If the timestamp is older than 15 minutes (900 seconds)
+    if [ "$time_diff" -gt 900 ]; then
+        # Dump logs
+        pid=$(pidof glow-monitor)
+        if [ -n "$pid" ]; then
+            kill -USR1 "$pid"
+        fi
```

Comment:
no need to dump logs for this restart

Reply:
maybe, if you really want, dump them to /dev/shm so they aren't hitting the disk at all but we can still look at them when we remote in, assuming the device hasn't reset

Reply:
I decided it would be good to have the report available, but we should definitely put it in `dev/shm` so that we aren't writing to disk every 15 minutes if the glow-monitor is having trouble. And, if you haven't seen yet, this script should only be restarting the glow_monitor service and not the entire device